### PR TITLE
Field email url restructure + minor UI fix

### PR DESCRIPTION
**Pull request by vittoriaThinkst**
## Proposed changes

Update all GenerateToken Form with the new structure
Update all validations for the new structure
Update Skeleton Loading component for GenerateToken Form

Examples:
Screenshot 2024-05-03 at 16 13 02
Screenshot 2024-05-03 at 16 10 32
Screenshot 2024-05-03 at 16 54 08
Screenshot 2024-05-03 at 16 53 37

Minor UI updates to the home:
Add Placeholder for headline  + subtitle
Fix animation of Card on hover: remove 'growing' effect, add light change on the background color
Fix mobile by removing the hover effect and adding a visual Button to generate Token
Remove the 'growing' card effect from Incident Alert list: keep the change on the border color
Fix Token names by removing 'Token' from their label

Tailwind: slightly changed the grey palette. What used to be a plain grey, has now a touch of green.
The difference is quite subtle ( I can see it on my Mac, I can't notice it on my other monitor )

Screenshot 2024-05-03 at 16 13 40
Screenshot 2024-05-03 at 16 13 20
Screenshot 2024-05-03 at 16 13 27

### CODE REVIEW

File: frontend_vue/src/components/ui/CardToken.vue [hunk 1]
```diff
@@ -89,11 +97,11 @@ function handleClickToken() {
 .token-card:hover + .token-card__documentation-link,
 .token-card:focus + .token-card__documentation-link,
 .token-card:focus-visible + .token-card__documentation-link {
-  top: -0.2em;
+  /* top: -0.2em; */
```
Comment (mclmax) [inline frontend_vue/src/components/ui/CardToken.vue:100]: remove?

File: frontend_vue/src/utils/tokenServices.ts [hunk 2]
```diff
@@ -208,7 +208,7 @@ export const tokenServices: TokenServicesType = {
       'Scan this QR Code with the WireGuard app on your phone or copy the config below.',
   },
   [TOKENS_TYPE.AZURE_ENTRA_CONFIG]: {
-    label: 'Azure Entra Config ID',
+    label: 'Azure Entra config ID',
```
Comment (mclmax) [inline frontend_vue/src/utils/tokenServices.ts:211]: is this the label we show on the front page? it should be `Azure Entra ID login`

**Review by mclmax — APPROVED** [review]

File: frontend_vue/src/components/ui/CardToken.vue (hunk 1 above)
Comment (vittoriaThinkst) [reply frontend_vue/src/components/ui/CardToken.vue:100]: woops, yeah I'll remove it! thank you!

File: frontend_vue/src/utils/tokenServices.ts (hunk 2 above)
Comment (vittoriaThinkst) [reply frontend_vue/src/utils/tokenServices.ts:211]: I fix it, thank you!

**Pull request merged**